components/telemetry-operator/controller/logpipeline/controller.go

```diff
@@ -107,6 +109,16 @@ func (r *Reconciler) doReconcile(ctx context.Context, pipeline *telemetryv1alpha
 		return err
 	}
 
+	if r.config.ManageFluentBit && pipeline.DeletionTimestamp.IsZero() {
+		config := resourceConfig{
+			BaseName:  fmt.Sprintf("telemetry-%s", r.config.DaemonSet.Name),
```

Comment:
Or do we just take one of the `fluent-bit-ds-name` or `fluent-bit-cm-name` since they have the same default values. Feels weird to introduce a third name arg that has the same default